feat: move to accelerate launch for distributed training (#92)
foundation-model-stack/fms-hf-tuning: examples/prompt_tuning_twitter_complaints/README.md

### Prompt Tuning
We will switch our PEFT method from LORA to Prompt Tuning (pt)
```bash
# Please set the environment variables:
# MASTER_PORT=1234 # The port at which the process with rank 0 listens to and should be set to an unused port
# MODEL_PATH=meta-llama/Llama-2-7b-hf # Huggingface model id or path to a checkpoint
# TRAIN_DATA_PATH=twitter_complaints.json # Path to the training dataset
# OUTPUT_PATH=out # Path to the output folder where the checkpoints are saved

accelerate launch \
--main_process_port $MASTER_PORT \
--config_file fixtures/accelerate_fsdp_defaults.yaml \
tuning/sft_trainer.py  \
--model_name_or_path $MODEL_PATH  \

> --training_data_path $DATA_PATH  \

Reviewer: ```suggestion
--training_data_path $TRAIN_DATA_PATH  \
```

DATA_PATH=twitter_complaints.json
OUTPUT_PATH=out

> # MASTER_PORT and MASTER_ADDR are essential for multi node training and 
> # not needed for multi gpu in single node
> MASTER_PORT=1234 # The port at which the process with rank 0 listens to
> MASTER_ADDR=x.x.x.x # The IP addresss of the node with rank 0
> 
> accelerate launch \
> --main_process_ip $MASTER_ADDR \
> --main_process_port $MASTER_PORT \
> --config_file fixtures/accelerate_fsdp_defaults.yaml \

Reviewer 2: ```suggestion
# MASTER_PORT should be set to an unused port
MASTER_PORT=1234 # The port at which the process with rank 0 listens to

accelerate launch \
--main_process_port $MASTER_PORT \
--config_file fixtures/accelerate_fsdp_defaults.yaml \
```
